src/executor/graph_executor.cc

```diff
@@ -562,16 +562,13 @@ void GraphExecutor::Init(nnvm::Symbol symbol,
                             arg_grad_ctxes, aux_state_ctxes, grad_req_types);
 
 #if MXNET_USE_NGRAPH == 1
-  // TODO(mbrookhart): Remove this when hetr can handle multiple contexts
-  auto multi_context = multi_context_check(default_ctx, in_arg_ctxes,
-                                           arg_grad_ctxes, aux_state_ctxes);
```

Comment:
We need to keep this is place, we can't handle multiple contexts, this will break other unit tests.

Reply:
As suggested, this change has been replaced by adding an env. variable MXNET_NGRAPH_GPU which toggles the conditional at L525 when set.